server/apps/uav.py

```diff
@@ -138,20 +138,6 @@ def uav_export_commands_file():
     return {"waypoints": waypoints}
 
 
-@uav_commands.route("/export")
```

Comment:
You sure this route is unnecessary?

Reply:
That route was a duplicate route (and also it was older and wrong). The correct export route should be right above it.